js/assets.ts

```diff
@@ -8,7 +8,7 @@
 import denoDts from "/js/deno.d.ts!string";
 // import libDts from "/third_party/node_modules/typescript/lib/lib.d.ts!string";
 import libDenoDts from "/js/lib.deno.d.ts!string";
-// import libDomD qts from "/third_party/node_modules/typescript/lib/lib.dom.d.ts!string";
+import libDomDts  from "/third_party/node_modules/typescript/lib/lib.dom.d.ts!string";
```

Comment:
Hmmm, what types came from the DOM that are needed?  That is a lot to pull in that isn't actually there.  Might make sense to just extract what is needed/supported.

Reply:
Just the TextEncoder and TextDecoder. I will give this commit another pass after #455 lands and ping you for review.

Maybe I will just use `@types/text-encoding` instead...

Reply:
No longer relying on lib.dom.d.ts now. We can't use @types/text-encoding due to this:
https://github.com/DefinitelyTyped/DefinitelyTyped/commit/f9a3f3c2da18980f7c5b089eb6d298dc0fd3797b#r30007625